packages/components/src/color-palette/README.md

```diff
@@ -44,12 +54,28 @@ classes to be applied to the container.
 
 ### clearable
 
-Whether the palette should have a clearing button or not.
+Whether the palette should have a clearing button.
 
 -   Type: `Boolean`
 -   Required: No
 -   Default: true
 
+### __experimentalHasMultipleOrigins
+
+Whether the colors prop is an array of color palettes, rather than an array of color objects.
+
+-   Type: `Boolean`
+-   Required: No
+-   Default: false
+
+### __experimentalIsRenderedInSidebar
+
+Whether this is rendered in the sidebar.
+
+-   Type: `Boolean`
+-   Required: No
+-   Default: false
+
```

Comment:
Let's actually not list experimental props in the README — apologies if I forgot to mention that earlier!

Reply:
Good idea, removed in https://github.com/WordPress/gutenberg/pull/44632/commits/4fd5d9c8b1fc2af1e70ce66c02506ed426463da3